builder/src/lib.rs

```diff
@@ -57,7 +57,7 @@ pub const FMC_WITH_UART: FwId = FwId {
 pub const APP_WITH_UART: FwId = FwId {
     crate_name: "caliptra-runtime",
     bin_name: "caliptra-runtime",
-    features: &["emu", "test_only_commands"],
+    features: &["emu", "test_only_commands", "fips_self_test"],
```

Comment:
Could you look at how big the `caliptra-runtime` image is with this feature enabled?

I usually just do `readelf -S path/to/caliptra-runtime` and then add the sizes of the `.text` and `.rodata` sections.

Reply:
According to the manifest, the size  of the RT binary image is 48276 bytes.

Reply:
Thanks!